src/shapes/object.class.js

```diff
@@ -981,6 +981,7 @@
         this.drawCacheOnCanvas(ctx);
       }
       else {
+        this._cacheCanvas = null;
```

Comment:
if we don't set this to null it's hard to know if the _cacheCanvas is up to date or not. I could imagine some kind of performance in memory by not keeping extra canvases in memory.